### PR TITLE
Updates for hawthorn upgrade

### PR DESCRIPTION
**Pull request by lgp171188**
Add changes from https://github.com/open-craft/openedx-deployment/commit/cca980973364bf1136640e5649dd93f31de26296#diff-40c660c4b60b66b30b41740d0dd05ea3

and applies upstream change: https://github.com/edx/configuration/pull/4912

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/analytics_sandbox.yml [hunk 1]
```diff
@@ -19,13 +19,11 @@
...
     - aws
     - security
-    - mysql
-    - edxlocal
-    - analytics_api
+    - common_vars
     - role: nginx
       nginx_sites:
         - insights
-        - analytics_api
```
Comment (lgp171188) [inline playbooks/analytics_sandbox.yml]: @pomegranited, this line was not present in https://github.com/open-craft/openedx-deployment/commit/cca980973364bf1136640e5649dd93f31de26296#diff-40c660c4b60b66b30b41740d0dd05ea3. So I have removed it.

Comment (pomegranited) [reply playbooks/analytics_sandbox.yml]: Yep that's ok -- epodx don't expose their analytics API to the web, they use SSH tunnels.

Comment (pomegranited) [reply playbooks/analytics_sandbox.yml]: Sorry, I spoke too soon..  we do need that `analytics_api` line to exist under `nginx_sites`, and as a role on its own.  

epodx *do* expose their analytics API via nginx, though they use SSH tunnels to restrict access to the port itself.  I tested this by following the instructions in the secure repo docs for [accessing the Analytics API](https://github.com/hks-epod/epodx/blob/master/docs/api.md#hks-epodx---epodx-analytics-api).

**Review by pomegranited — CHANGES_REQUESTED** [review]
See [comment about analytics_api](https://github.com/hks-epod/configuration/pull/6/files#r243488242).

File: playbooks/analytics_sandbox.yml (hunk 1 above)
Comment (lgp171188) [reply playbooks/analytics_sandbox.yml]: @pomegranited, I will fix this and redeploy insights after the LMS redeployment is done and everything is working. :+1:

**Review by pomegranited — APPROVED** [review]
:+1: Thanks @lgp171188 !

- [x] I tested this by verifying the newly deployed analytics and edxapp instances are running the required services, and deployed successfully.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A, changes come from other PRs.

**Pull request merged**